tutorials/plot_object_annotations.py

```diff
@@ -189,6 +180,47 @@
 print(annot)
 
 ###############################################################################
+# Iterating, Indexing and Slicing :class:`mne.Annotations`
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# :class:`~mne.Annotations` supports the same indexing and slicing as
+# ``numpy.array``, and it returns a ``(onset, duration, description)`` tuple
+# with the sliced elements.
+#
+# See the following examples and usages:
+plt.close('all')
+print('type: {0} content: {1}'.format(type(annot[0]), annot[0]))
+
+print('type: {0} content: {1}'.format(type(annot[:]), annot[:]))
+
+_MSG = 'annotation #{0}: onset={1} duration={2} desc={3}'
+for idx, (onset, duration, desc) in enumerate(annot):
+    print(_MSG.format(idx, onset, duration, desc))
+
+start, stop, step = (0, None, 2)
+every_other_annotation = slice(start, stop, step)
+for onset, duration, desc in zip(*annot[every_other_annotation]):
+    print('onset={0} duration={1} desc={2}'.format(onset, duration, desc))
```

Comment:
I would try to dumb it down. Here you use advanced string formatting, for loops etc. I would just do:

```
annotations[:3]   # will return a new Annotations formed by the first 3
annotations[2]   # will return a new Annotations restricted to the 3rd annotation.
```

and I would point to python indexing doc as it behaves likes for str or lists etc.

Reply:
Thats exactly what does not wat I expected from the tests.